# Errro Handling 
# try , except , else , finally
print("hello")
# a = 10
try:
    def abcd(c,b):
        return c+b
    print(abcd(1,1))
except TypeError:
    print("Error")
else:
    print("No error")
finally:
    print("finally block")
print("hello")

